stellargraph/mapper/mini_batch_node_generators.py

```diff
@@ -251,12 +260,10 @@ def __init__(
                     "When passed together targets and indices should be the same length."
```

Comment:
`targets is not None` (above) must already be true, since it was tested previously.

Reply:
Good catch!